draft-dnsop-deleg-transport.md

```diff
@@ -182,8 +188,8 @@ To avoid a circular dependency, "tlsa" MUST appear in any DELEG record that is u
 Resolvers that support TLS-based transports MUST adopt one of the following behaviors:
 
 1. Use DANE for authentication, and treat any endpoints lacking DANE support as incompatible.
-1. Use PKI for authentication, and treat any DANE-only endpoint as incompatible.  In this behavior, compatibility with `tlsa=disabled` endpoints is REQUIRED.
-1. Support both DANE and PKI for authentication, preferring DANE if it is available for each endpoint.
+2. Use PKI for authentication, and treat any DANE-only endpoint as incompatible.  In this behavior, compatibility with `tlsa=disabled` endpoints is REQUIRED.
+3. Support both DANE and PKI for authentication, preferring DANE if it is available for each endpoint.
```

Comment:
Markdown auto-increments numeric lists, so the numbers here have no effect.

Reply:
I thought so but I was editing on a plane and had no formatter so couldn't confirm.  Just out of curiosity, if you have numbered them but done so (apparently) incorrectly, does it fix them?  eg, 1, 2, 2, 4 ...

Reply:
Yep, it completely ignores the actual numbers in the source code.  (This makes me sad: I want to be able to start with 0!)